msgpack/ext.py

```diff
@@ -5,17 +5,7 @@
 import struct
 
 
-PY2 = sys.version_info[0] == 2
-
-if PY2:
-    int_types = (int, long)
-    _utc = None
-else:
-    int_types = int
-    try:
-        _utc = datetime.timezone.utc
-    except AttributeError:
-        _utc = datetime.timezone(datetime.timedelta(0))
+_utc = datetime.timezone.utc
```

Comment:
Could you remove this?

Reply:
Oops, I forgot to remove the line. It's done now.